<launch>
  <!-- By default, we are not in debug mode -->
  <arg name="debug" default="false" />

  <!-- static tf publishers -->
  <node name="camera_link_publisher" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 map camera_link 100" />
  <node name="openni_depth_frame_publisher" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 camera_depth_frame openni_depth_frame 100" />

  <!-- future obstacle publisher -->
  <node name="future_obstacle_publisher" pkg="pcml" type="future_obstacle_publisher" output="screen">
    <param name="input_stream_type" value="realtime"/>
    <param name="joints_type" value="upper_body"/>
    <param name="render" value="true"/>

    <!-- CAD120 parameters-->
    <!-- <param name="cad120_directory" value="/playpen/jaesungp/dataset/CAD120"/> -->
  </node>

  <!-- future obstacle visualizer -->
  <node name="future_obstacle_visualizer" pkg="pcml" type="future_obstacle_visualizer" output="screen">
  </node>

  <!-- rviz will not be opened automatically -->
</launch>
